ding2.install

```diff
@@ -372,6 +372,21 @@ function ding2_update_7021() {
  */
 function ding2_update_7022() {
   module_enable(array('ding_p2_installer'));
+
```

Comment:
This should be place in it own update function as environments may have been upgraded already, hence this may not be executed.